auditbeat/module/file_integrity/event.go

```diff
@@ -20,6 +20,8 @@ import (
 	"golang.org/x/crypto/blake2b"
 	"golang.org/x/crypto/sha3"
 
+	"github.com/OneOfOne/xxhash"
```

Comment:
Normally we have three import groups (stdlib, non-elastic, then elastic). So just merge this one into the group above.